homeassistant/components/velux/cover.py

```diff
@@ -46,6 +46,11 @@ async def async_added_to_hass(self):
         """Store register state change callback."""
         self.async_register_callbacks()
 
+    @property
+    def unique_id(self):
+        """Return the unique ID of this cover."""
+        return f"{DOMAIN}-cover_{self.node.serial_number}"
```

Comment:
if im not mistaken, the unique id should already be unique relative to domain. So it shouldnt need to part of the unique id here

Im not familiar with velux but does it consist of multiple nodes? and a single host? ie could there be multiple covers?

Reply:
Your right. It is sufficient to just use the serial number, I wasn't aware the domain is already part of it. 
Fixed it.